examples/llm/inference/convert_composer_to_hf.py

```diff
@@ -0,0 +1,180 @@
+        loaded_model = transformers.AutoModelForSequenceClassification.from_pretrained('./hf-save-pretrained-output')
+
+    Args:
+        checkpoint_path (Union[Path, str]): Path to the composer checkpoint, can be a local path, or a remote path beginning with ``s3://``, or another backend
+            supported by :meth:`composer.utils.maybe_create_object_store_from_uri`.
+        output_folder (Union[Path, str]): Path to the folder to write the output to. Must be a local path.
+        local_checkpoint_save_location (Optional[Union[Path, str]], optional): If specified, where to save the checkpoint file to locally.
+                                                                                If the input ``checkpoint_path`` is already a local path, this will be a symlink.
+                                                                                Defaults to None, which will use a temporary file.
+    """
+    # default local path to a tempfile if path is not provided
+    if local_checkpoint_save_location is None:
+        tmp_dir = tempfile.TemporaryDirectory()
+        local_checkpoint_save_location = Path(
+            tmp_dir.name) / 'local-composer-checkpoint.pt'
+
+    # download the checkpoint file
+    get_file(str(checkpoint_path), str(local_checkpoint_save_location))
+
+    # Load the Composer checkpoint state dict
+    composer_state_dict = safe_torch_load(local_checkpoint_save_location)
+
+    # Build and save HF Config
+    print('Saving HF Model Config...')
+    hf_config = get_hf_config_from_composer_state_dict(composer_state_dict)
+    hf_config.save_pretrained(output_folder)
+    print(hf_config)
+
+    # Extract and save the HF tokenizer
+    print('#' * 30)
+    print('Saving HF Tokenizer Config...')
+    hf_tokenizer = get_hf_tokenizer_from_comopser_state_dict(
+        composer_state_dict)
+    hf_tokenizer.save_pretrained(output_folder)
+    print(hf_tokenizer)
+
+    # # Extract and save the HF model weights
+    print('#' * 30)
+    print('Saving HF Model Weights...')
+    weights_state_dict = composer_state_dict['state']['model']
+    weights_state_dict = {
+        k: v for k, v in weights_state_dict.items() if k.startswith('model.')
+    }
+    torch.nn.modules.utils.consume_prefix_in_state_dict_if_present(
+        weights_state_dict, prefix='model.')
+    torch.save(weights_state_dict, Path(output_folder) / 'pytorch_model.bin')
+
+    print(f'HF checkpoint folder successfully created at {output_folder}.')
+    print('#' * 30)
+
+
+def parse_args() -> Namespace:
+    """Parse commandline arguments."""
+    parser = ArgumentParser(
+        description=
+        'Convert Composer checkpoint and Omegaconf model config into a standard HuggingFace checkpoint folder.'
+    )
+    parser.add_argument('--composer_ckpt', type=str, required=True)
+    parser.add_argument('--hf_folder', type=str, required=True)
```

Comment:
```suggestion
    parser.add_argument('--hf_output_folder', type=str, required=True)
```

Reply:
I'm thinking of making it `hf_path` because it can support URI now

Reply:
ok then `--hf_output_path` :D